dataland-batch-manager/src/main/resources/application-development.properties

```diff
@@ -2,5 +2,6 @@ dataland.dataland-batch-manager.client-id=dataland-batch-manager
 dataland.dataland-batch-manager.client-secret=password
 dataland.dataland-batch-managet.get-all-gleif-companies.force=false
 dataland.dataland-batch-managet.get-all-gleif-companies.flag-file=./this-file-does-not-exist
+dataland.dataland-batch-manager.mapping-file=./deltafiles/isinMapping.csv
```

Comment:
This should be an absolute path as well.

Reply:
removed it entirely